bokehjs/src/coffee/tool/gestures/box_select_tool.coffee

```diff
@@ -62,10 +62,34 @@ class BoxSelectToolView extends SelectTool.View
       sm = ds.get('selection_manager')
       sm.select(@, @plot_view.renderers[r.id], geometry, final, append)
 
+    if @mget('callback')?
+      @_emit_callback(geometry)
+
     @_save_geometry(geometry, final, append)
 
     return null
 
+  _emit_callback: (geometry) ->
+    r = @mget('renderers')[0]
```

Comment:
This is just a question for my own learning - why do you take the first renderer here?